README.md

```diff
@@ -70,31 +70,21 @@ Global installation is supported as well, but not recommended.
 
 ### Development
 
-Rebuild the Panel plugin on any file changes:
+Start a development server for the panel plugin:
```

Comment:
The Kirby Panel is a proper noun, that's why I uppercased it. Let's keep it that way, shall we?